test/regressions/vite.config.mts

```diff
@@ -0,0 +1,76 @@
+import { defineConfig, transformWithEsbuild } from 'vite';
+import react from '@vitejs/plugin-react';
+import * as path from 'path';
+import * as url from 'url';
+
+const currentDirectory = url.fileURLToPath(new URL('.', import.meta.url));
+const WORKSPACE_ROOT = path.resolve(currentDirectory, '../../');
+
+// https://vite.dev/config/
+export default defineConfig({
+  plugins: [
+    {
+      // Unfortunatelly necessary as we opted to write our jsx in js files
```

Comment:
🙈 🤷

Reply:
It's on my to do list. This has been causing trouble every time we try to introduce a tool that relies on esbuild.